CHANGES.md

```diff
@@ -52,6 +52,40 @@
 
 * ([#X](https://github.com/apache/beam/issues/X)).
 -->
+# [2.56.0] - Unreleased
+
+## Highlights
+
+* New highly anticipated feature X added to Python SDK ([#X](https://github.com/apache/beam/issues/X)).
+* New highly anticipated feature Y added to Java SDK ([#Y](https://github.com/apache/beam/issues/Y)).
+
+## I/Os
+
+* Support for X source added (Java/Python) ([#X](https://github.com/apache/beam/issues/X)).
+
+## New Features / Improvements
+
+* X feature added (Java/Python) ([#X](https://github.com/apache/beam/issues/X)).
+
+## Breaking Changes
+
+* X behavior was changed ([#X](https://github.com/apache/beam/issues/X)).
+
+## Deprecations
+
+* X behavior is deprecated and will be removed in X versions ([#X](https://github.com/apache/beam/issues/X)).
+
+## Bugfixes
+
+* Fixed locking issue when shutting down inactive bundle processors. Symptoms of this issue include slowness or stuckness in long-running jobs (Python) ([#30679](https://github.com/apache/beam/pull/30679)).
```

Comment:
thanks, does the locking cause slowness or also stuckness? What would cause stuckness?

Reply:
It definitely seemed to cause slowness (we saw lock contention for this lock in a very slow job). Stuckness may not be quite true, but in this case at least it was slow enough that it was perceived as stuckness at points